charts/alertmanager/templates/alertmanager-sidecar.yaml

```diff
@@ -0,0 +1,41 @@
+################################
+## Alertmanager sidecar ConfigMap
+#################################
+{{- if .Values.global.authSidecar.enabled }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "alertmanager.fullname" . }}-nginx-conf
+  labels:
+    tier: monitoring
+    component: {{ template "alertmanager.name" . }}
+    chart: {{ template "alertmanager.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  default.conf: |
+    upstream astro-alertmanager {
+      server {{ .Release.Name }}-alertmanager:{{ .Values.ports.http }} ;
```

Comment:
And don't go back through the service here, use `localhost` instead.

Also, I feel like this filename should be `alertmanager-sidecar-configmap.yaml`, or be moved into `alertmanager-configmap.yaml`?

Reply:
@jedcunningham  both changes are added.

Reply:
I don't see the filename change?

Reply:
Looks good.